astropy/units/si.py

```diff
@@ -20,7 +20,7 @@
 
 def_unit(
     ["percent", "pct"],
-    Unit(0.01),
```

Comment:
This caused the import loop I mentioned in https://github.com/astropy/astropy/pull/17355#discussion_r1840671746. Turns out that there weren't any others, but I couldn't know that because Python errors out as soon as it detects the first one. Also, our practice is not to backport code changes like this here, so it would have been beyond the scope in the other pull request anyways.